### PR TITLE
Update seed data

### PR DESCRIPTION
**Pull request by jnaoroji**
modified swapData.json and server.js, modifed seed data

### CODE REVIEW

File: seeds/swapData.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "title": "Strawberry Seedlings",
+    "description": "Many stawberry seedlings to swap, will swap for other berries or fruit",
+    "status": "Swap"
+  },
```
Comment (Natasa00) [inline seeds/swapData.json]: This is missing `date_created` and `user_id` fields.
Make sure your date field is in ISO format `'2023-02-03T15:05:00.234Z'`

File: package.json [hunk 2]
```diff
@@ -13,13 +13,17 @@
...
+    "and": "^0.0.3",
     "bcrypt": "^5.0.0",
     "connect-session-sequelize": "^7.0.4",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "express-handlebars": "^5.2.0",
     "express-session": "^1.17.1",
     "mysql2": "^2.2.5",
+    "npm": "^9.7.2",
+    "run": "^1.4.0",
```
Comment (Natasa00) [inline package.json:25]: why do we need npm and run as dependencies? We should remove these

**Review by Natasa00 — CHANGES_REQUESTED** [review]

File: models/Swap.js [hunk 3]
```diff
@@ -15,6 +15,11 @@ Swap.init(
       type: DataTypes.STRING,
       allowNull: false,
     },
+    status: {
```
Comment (Natasa00) [inline models/Swap.js]: We should rename this to `type`

**Review by mcramileux — APPROVED** [review]

File: seeds/swapData.json (hunk 1 above)
Comment (jnaoroji) [reply seeds/swapData.json]: Yes mate, those will be auto incremented.

File: models/Swap.js (hunk 3 above)
Comment (jnaoroji) [reply models/Swap.js]: ok

File: seeds/swapData.json (hunk 1 above)
Comment (Natasa00) [reply seeds/swapData.json]: @jnaoroji No the `id` of the Swap is auto-incremented, not the `date_created` and `user_id` which are both mandatory fields per your model definition.

**Review by Natasa00 — APPROVED** [review]
approved

**Review by Natasa00 — APPROVED** [review]
approved

**Pull request merged**